crates/astria-sequencer/src/ibc/ics20_withdrawal.rs

```diff
@@ -105,14 +107,14 @@ async fn create_ibc_packet_from_withdrawal<S: StateRead>(
 /// 1. Errors reading from DB
 /// 2. `action.bridge_address` is set, but `from` is not the withdrawer address.
 /// 3. `action.bridge_address` is unset, but `from` is a bridge account.
-async fn establish_withdrawal_target<S: StateRead>(
-    action: &action::Ics20Withdrawal,
+async fn establish_withdrawal_target<'a, S: StateRead>(
+    action: &'a action::Ics20Withdrawal,
     state: &S,
-    from: [u8; 20],
-) -> Result<[u8; 20]> {
+    from: &'a [u8; 20],
+) -> Result<&'a [u8; 20]> {
```

Comment:
The lifetime `'a` might cause confusion on refactors.

Reply:
It seems reasonably innocuous to me.  I'd prefer to leave as is to avoid copying the bytes needlessly.